examples/common.py

```diff
@@ -0,0 +1,3 @@
+from tianshou.highlevel.logger import LoggerFactoryDefault
+
+logger_factory = LoggerFactoryDefault()
```

Comment:
I'm merging this PR, but for the next PR: this introduces a mutable default throughout the code. Why was this neccessary?

Reply:
> Why was this necessary?

The examples create a logger using the same code that `LoggerFactoryDefault.create_logger` already implements.

To be DRY, my idea was to re-use the LoggerFactory instead of duplicating the code.

Furthermore, I thought it is cleaner to have one single factory where each example is re-using it to create its own logger.

One alternative solution, using this approach, is to instantiate a `LoggerFactoryDefault` per example file.

Reply:
One instantiation per file sounds good to me

Reply:
Resolved, cf. https://github.com/thu-ml/tianshou/pull/1077#discussion_r1547404024.